src/containerapp/azext_containerapp/_up_utils.py

```diff
@@ -1214,8 +1214,8 @@ def _get_env_and_group_from_log_analytics(
                 resource_group.name = env_details["resource_group"]
 
 
-def _get_acr_from_image(cmd, app):
-    if app.image is not None and "azurecr.io" in app.image:
+def _get_acr_from_image(cmd, app, registry_server_set=None):
```

Comment:
Please rerun tests in `test_containerapp_up.py` lively with `--live`

Reply:
updated test recording in the latest commit. Thanks!